setup.cfg

```
@@ -1,60 +1,139 @@
 [metadata]
 name = sunkit_instruments
-author = The SunPy Developers
+provides = sunkit_instruments
+description = "A sunpy-affiliated package for solar instrument-specific tools."
```

```suggestion
description = "A SunPy-affiliated package for solar instrument-specific tools."
```

It's the project name so it should be capitalized I think?